### ISSUE TITLE
Xgrid.set_ylabels and Ygrid.set_ylabels ignore default font size

### ISSUE THREAD

**Issue by jsegu** [matplotlib/trendvis#27]
Xgrid.set_ylabels and Ygrid.set_ylabels use a default of 14 points. Without the user specifying a fontsize, this will often result in different font sizes for x and y labels. Could this default be replaced by the matplotlib default from rcParams?

```python
>>> import matplotlib.pyplot as plt
>>> import trendvis
>>> plt.rcParams['font.size'] = '10.0'
>>> grid = trendvis.XGrid([1,2,1])
>>> grid.set_ylabels(['ylabel']*3)
>>> print grid.axes[-1][-1].yaxis.get_label().get_fontsize()
14.0
```

Thanks!

**Comment by mscross** [comment]
I support this change.  Another PR, @jsegu ?

**Comment by mscross** [comment]
Closed by #28
